src/components/ShortcutLink/styled.ts

```diff
@@ -0,0 +1,39 @@
+import { usingColors } from 'src/hooks/theme';
+import styled from 'styled-components';
+
+export const ShortcutLinkWrapper = styled.a`
+  margin: 16px 32px;
+  display: flex;
+  flex-direction: column;
+  & > * {
+    color: ${usingColors((t) => t.ON.SECONDARY_ACCENT)};
+    width: 320px;
+    height: 50px;
```

Comment:
Remove the & > * selector, and move the contents into the ShortcutLinkField styled component.